README.md

```diff
@@ -57,6 +57,22 @@ build gossamer command:
 ```
 make gossamer
 ```
+### Troubleshooting for Apple Silicon users
+
+If you are facing the following problem with the `wasmer`:
+````
+undefined: cWasmerImportObjectT
+undefined: cWasmerImportFuncT
+undefined: cWasmerValueTag
+````
+Make sure you have the following Golang enviroment variables:
+
+1. GOARCH="amd64"
+2. CGO_ENABLED="1"
```

Comment:
nit let's use bullet points since there is no order importance

We can also remove the double quotes in this case

```suggestion
- GOARCH=amd64
- CGO_ENABLED=1
```

Reply:
Done!